pkg/bootstrap/docker/up.go

```diff
@@ -334,8 +335,18 @@ func (c *ClientStartConfig) GetDockerClient(out io.Writer) error {
 	if err != nil {
 		return errors.ErrNoDockerClient(err)
 	}
+	// FIXME: Workaround for docker engine API client on OS X - sets the default to
+	// the wrong DOCKER_HOST string
+	if runtime.GOOS == "darwin" {
+		dockerHost := os.Getenv("DOCKER_HOST")
+		if len(dockerHost) == 0 {
+			os.Setenv("DOCKER_HOST", "unix:///var/run/docker.sock")
```

Comment:
Yes, if you are running the Docker beta.

If you are not running Docker beta, then your DOCKER_HOST needs to be set for you to connect anyway. The problem with the engine-api client right now is that it's defaulting to tcp://localhost:2375 when DOCKER_HOST is not set on the Mac. That only works on Windows.